nansat/mappers/mapper_aapp_l1b.py

```diff
@@ -10,11 +10,15 @@
 import sys
 import struct
 import datetime
+import logging
 
 from nansat.exceptions import WrongMapperError
 from nansat.geolocation import Geolocation
 from nansat.vrt import VRT
 
+LOGGER = logging.getLogger("Nansat."+__name__)
```

Comment:
`__name__` already contains the full name of the module (here, it is`nansat.mappers.mapper_aapp_l1b`), so there is no need to prefix it with "Nansat.".
Same thing for the other mappers.

Reply:
yes, you are right! But, when we run it from harvesting, only `mapper_aapp_l1b ` is appear! It is better to keep that in the name of the lagger,isn't it?

Reply:
Then write the full module name (`nansat.mappers.mapper_aapp_l1b`), that way there is no ambiguity.

Reply:
Done